tests/Throttling.txt

```diff
@@ -1,10 +1,10 @@
 testTitle=Throttling
     testName=Throttling
     testDuration=60.0
-    healthMetricsCheckInterval=1.0
     actorsPerClient=10
     readsPerTransaction=10
     writesPerTransaction=10
     throttlingMultiplier=0.5
-    sendDetailedHealthMetrics=true
     maxBurst=10000
+
+    testName=HealthMetricsApi
```

Comment:
You'll have to rebase or merge with master, because this test file will need to be converted to toml.